src/interfaces/store-interfaces/state-interfaces/registration-state-interface.ts

```diff
@@ -15,4 +15,5 @@ export interface RegistrationStateIF {
   naics: NaicsIF
   businessType: BusinessTypes
   businessNumber?: string // GPs only
+  businessTypeConfirm?: boolean
```

Comment:
As this is no longer an optional field, please remove the `?`.

Reply:
I removed ? in my latest commit: c77ff41.